javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -629,6 +593,7 @@ public Object slang_persistent() {
         }
     }
 
+    /** Add all elements (one-by-one, as we're not testing bulk operations) */
```

Comment:
@RichardWarburton, we're using loops here, as we're simulating rebuilding the given data structure, one-by-one.
We're interested in the amortized cost of the operations across different edge cases (e.g. where append needs to add a new level to the tree).
If we could use bulk operations, we could simply return the already created instance.
Does it make sense or do you recommend a better approach :)?